docs/src/man/comparisons.md

```diff
@@ -235,6 +235,58 @@ The table below compares more advanced commands:
 | DataFrame as input        | `summarize(df, head(across(), 2))`                        | `combine(d -> first(d, 2), df)`                               |
 | DataFrame as output       | `summarize(df, tibble(value = c(min(x), max(x))))`        | `combine(:x => x -> (value = [minimum(x), maximum(x)],), df)` |
 
+## Comparison with the R package data.table
+
+The following table compares the main functions of DataFrames.jl with the R package data.table (version 1.14.1). To contrast data.table with dplyr, the focus has been put on transformations in place.
+
+|Operations              | data.table  | DataFrames.jl|  
+|:-----------------------|:------------|:-------------|
+|Reduce multiple values  |`df[, list(mean(x))]`       |`combine(df, :x => mean)`|
+|Add new columns         |`df[, x_mean := mean(x) ]`  |`transform(df, :x => mean => :x_mean)`|
+|Rename column (in place) |`setnames(df, "x", "x_new")`|`rename!(df, :x => :x_new)`|
+|Rename multiple columns (in place) |`setnames(df, c("x", "y"), c("x_new", "y_new")`|`rename!(df, [:x, :y] .=> [:x_new, :y_new])`|
```

Comment:
Not sure whether it fits, but it would be nice to show the more idiomatic approach:
```suggestion
|Rename multiple columns (in place) |`setnames(df, c("x", "y"), c("x_new", "y_new")`|`rename!(df, :x => :x_new, :y => :y_new)` or `rename!(df, [:x, :y] .=> [:x_new, :y_new])`|
```

Reply:
Correct me if I am wrong. The `setnames` function in data.table is nice because it is convenient for programmatic use. `old_var = ...; new_var = ...; setnames(df, old_var, new_var)`.

I am thinking that the form I have in the text is easiest to use in this way?

Reply:
In DataFrames.jl both forms are easy enough to use programatically (but I agree that vector form is more natural and it would have lower compilation latency).
In fact `rename!(df, :x => :x_new, :y => :y_new)` can be obtained from your form by writing `rename!(df, ([:x, :y] .=> [:x_new, :y_new])...)`.

Reply:
Yeah it really depends on the use case, which is why showing both sounds useful if that fits.

Reply:
But since the single-argument case is shown above maybe that's enough and people will guess that they can pass multiple pairs if needed.